src/lib/Behat/UtilityContext.php

```diff
@@ -34,18 +38,82 @@ public function waitUntilElementIsVisible(string $cssSelector, int $timeout = 5)
         }
     }
 
```

Comment:
I've added methods from StudioUI that I found useful + adapted existing ones a bit.